examples/jax/cpu-demo/demo.yaml

```diff
@@ -12,7 +12,7 @@ spec:
         spec:
           containers:
             - name: jax
-              image: docker.io/sandipanify/jaxgoogle:latest
+              image: docker.io/kubeflow/jaxjob-simple:latest
```

Comment:
I confirmed that there is this image in the DockerHub: https://hub.docker.com/r/kubeflow/jaxjob-simple/tags